crates/uv/src/commands/project/init.rs

```diff
@@ -99,6 +109,16 @@ pub(crate) async fn init(
         None => {
             writeln!(printer.stderr(), "Initialized project `{}`", name.cyan())?;
         }
+        // Initialized script
+        Some(path) if project_kind == InitProjectKind::Script => {
```

Comment:
Perhaps we should have a match on `project_kind` above this? If we change something around path handling in the future, we could easily miss the `Script` case with this structure.